queries.sql

```diff
@@ -49,3 +49,30 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) AS escape_attempts FROM animals GROUP BY neutered;
 SELECT MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+/* What animals belong to Melody Pond? */
+SELECT animals.name
+FROM animals
+INNER JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name
+FROM animals
+INNER JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
+
+/* How many animals are there per species? */
+SELECT species.name, COUNT(animals.id) AS animal_count
+FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.name;
+
```

Comment:
- [ ] Great job adding the queries. At the moment, you are missing 2 queries as per project requirements. See screenshot below and add these queries. But you have the screenshots of these queries but absent in your queries.sql file
![image](https://github.com/danielochuba/vet-clinic-database/assets/62849872/c01e8478-821e-4ecd-8f11-08cf810ca341)